astroquery/conftest.py

```diff
@@ -57,3 +60,14 @@ def pytest_addoption(parser):
         help='ALMA site (almascience.nrao.edu, almascience.eso.org or '
              'almascience.nao.ac.jp for example)'
     )
+
+
+@pytest.fixture(scope='function')
+def tmp_cwd(tmp_path):
+    """Perform test in a pristine temporary working directory."""
```

Comment:
Apparently this is needed at the moment, but the ability to set the output directory for all modules would be even better.

Reply:
please do open a tracking issue for this as it's beyond the scope of the current PR

Reply:
Yes, I'll open an issue and point to the locations we have to use `tmp_cwd` in the tests, so that the astroquery module can be a bit better about making locations one can dump files configurable.

Reply:
I would say this would be a UX enhancement for the user API (and thus the scope being outside of this current PR), more than convenience for us for testing. (I hate when I need to be in the right directory to run a line of code)

Reply:
Added #2528